neqo-transport/src/connection/mod.rs

```diff
@@ -225,6 +225,69 @@ impl AddressValidationInfo {
     }
 }
 
+#[derive(Clone)]
+pub struct ConnectionParameters {
+    quic_version: QuicVersion,
+    cc_algorithm: CongestionControlAlgorithm,
+    max_streams_bidi: u64,
+    max_streams_uni: u64,
+}
+
+impl Default for ConnectionParameters {
+    fn default() -> Self {
+        Self {
+            quic_version: QuicVersion::default(),
+            cc_algorithm: CongestionControlAlgorithm::NewReno,
+            max_streams_bidi: LOCAL_STREAM_LIMIT_BIDI,
+            max_streams_uni: LOCAL_STREAM_LIMIT_UNI,
+        }
+    }
+}
+
+impl ConnectionParameters {
+    pub fn get_quic_version(&self) -> QuicVersion {
+        self.quic_version
+    }
```

Comment:
Do you think we need these getters?  You haven't used these consistently, and it might be enough that we can access the members directly.

Reply:
I will leave them. All the members are private, therefore we need them.